src/fundamental/string-util-fundamental.c

```diff
@@ -77,19 +77,18 @@ sd_char* endswith_no_case(const sd_char *s, const sd_char *postfix) {
         return (sd_char*) s + sl - pl;
 }
 
-#ifdef SD_BOOT
-static sd_bool isdigit(sd_char a) {
+static sd_bool is_digit(sd_char a) {
```

Comment:
maybe we should use a more explicit name. The underscore is not a good indicator that this is strictly doing ASCII stuff. hence, following the logic of ascii_toupper(), ascii_strlower(), ascii_strcasecmp(), we should call this helper ascii_isdigit() I think.

Reply:
and probably export it, in fact.